cmake/RpMalloc.cmake

```
@@ -0,0 +1,32 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+include(FetchContent)
+FetchContent_Declare(
+        rpmalloc
+        URL      https://github.com/mjansson/rpmalloc/archive/refs/tags/1.4.4.tar.gz
+        URL_HASH SHA256=3859620c03e6473f0b3f16a4e965e7c049594253f70e8370fb9caa0e4118accb
+)
+FetchContent_GetProperties(rpmalloc)
+
+if(NOT rpmalloc_POPULATED)
+    FetchContent_Populate(rpmalloc)
+    add_library(rpmalloc ${rpmalloc_SOURCE_DIR}/rpmalloc/rpmalloc.c)
+    target_include_directories(rpmalloc PUBLIC ${rpmalloc_SOURCE_DIR}/rpmalloc)
+    target_compile_definitions("rpmalloc" PRIVATE ENABLE_OVERRIDE=1 ENABLE_PRELOAD=1)
```

The quotes are unnecessary
```suggestion
    target_compile_definitions(rpmalloc PRIVATE ENABLE_OVERRIDE=1 ENABLE_PRELOAD=1)
```

---

:+1: https://github.com/apache/nifi-minifi-cpp/pull/1456/commits/d08d3c78709c3a1042429de7ac18591909f6e4d2#diff-99683856935c1528793ea042b7f6ef4df3b5671a01b981f2c09dc928653cfea4R31